src/Item/ViewItem/ItemAcquisition/useItemAcquisition.js

```diff
@@ -68,6 +83,8 @@ const useItemAcquisition = (id) => {
         stripes.hasInterface('order-lines') &&
         stripes.hasInterface('orders') &&
         stripes.hasInterface('organizations.organizations') &&
+        stripes.hasInterface('finance') &&
```

Comment:
Please add these new interfaces to `optionalOkapiInterfaces` in `package.json`

Reply:
Added `"acquisition-methods": "1.0"` and `"finance.transactions": "6.0"` to `optionalOkapiInterfaces`